### ISSUE TITLE
One Stood Against Many without 3 targets should leave the creature readied.

### ISSUE THREAD

**Issue by tuism** [jeremylarner/keyteki#234]
As topic. If there are no more valid targets for the creature after One Stood Against Many, the creature should end the action ready.

**Comment by G0dr1c** [comment]
Not true.  So if only one card is available to fight, then it will fight with just that one.  Since there is not another target to fight, it would not ready.  This is different than an Anger ability because of the clause "different enemy creature".
